nextflow_schema.json

```diff
@@ -5,6 +5,83 @@
     "description": "IRIDA Next Example Pipeline",
     "type": "object",
     "definitions": {
+        "arborview": {
+            "title": "ArborView",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "av_html": {
+                    "type": "string",
+                    "default": "./assets/ArborView.html",
+                    "hidden": true
```

Comment:
Is this here in case someone wants to provide an `ArborView.html` as input?

Reply:
Never! I imagine this should be set in the modules.config then?

Reply:
Possibly? I guess I'm confused what is happening here. Haha.

Reply:
I did not adjust this, but I did not forget about it! Ill make the changes in a new PR